v_0_1_0/master_template.go

```diff
@@ -1803,7 +1805,7 @@ coreos:
       RestartSec=0
       TimeoutStopSec=10
       EnvironmentFile=/etc/network-environment
-      Environment="IMAGE={{.Cluster.Kubernetes.Hyperkube.Docker.Image}}"
+      Environment="IMAGE=quay.io/giantswarm/hyperkube:v1.8.1_coreos.0"
```

Comment:
A constant for this would be super useful. We should maybe have a `version.go` in each folder which contains such information to have one place we can look at. I will think about this a bit more.

Reply:
This make sense to me, we can update  much easier then

Reply:
true, that would be cool, at least for hyperkube I would welcome this, good thinking @xh3b4sd <3

Reply:
<3